src/content/admin/registry-credentials/amazon-ecr.mdx

```diff
@@ -5,74 +5,57 @@ sidebar_label: Amazon ECR
 id: amazon-ecr
 ---
 
-This tutorial will guide you on how to use ECR private registry in Amazon Web Services with Okteto and how to configure its necessary credentials.
-It's recommended that you have the [AWS CLI installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions) for this configuration.
+This tutorial will guide you on how to use your private Elastic Container Registry (ECR) with Okteto.
+It's recommended that you have the [AWS CLI installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions) to follow this tutorial.
 
-The steps to do it are: 
+The steps to configure your private ECR with Okteto are: 
 
-1. Create User with access to ECR
-1. Get User Credentials
-1. Okteto config with credentials
-1. Add credentials to the Okteto Dashboard
+- Create a user with access to your private ECR
+- Retrieve the user credentials
+- Configure the credentials in Okteto
 
-## Step 1: Create User with access to ECR
+## Step 1: Create a user with access to your private ECR
 
-Create IAM user with `aws cli` ([CLI Reference](https://docs.aws.amazon.com/cli/latest/reference/iam/create-user.html))
+Create IAM user with the [AWS CLI](https://docs.aws.amazon.com/cli/latest/reference/iam/create-user.html) by executing:
 
-```
+```bash
 aws iam create-user --user-name private-registry-user
+```
 
+```bash
 aws iam attach-user-policy \
---policy-arn arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly \
---user-name private-registry-user
+  --policy-arn arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly \
+  --user-name private-registry-user
 ```
 
-Or, configure manually. Find the `IAM Service` in the **AWS Console,** and follow these steps:
-
-- *Users → Create User*
-- Add User *Name*
-- Add *Policy Option*
-- Search for *ContainerRegistry* (for this example I’m adding **ReadOnly**)
-- Create User
-
-:::info
-It is recommended to use groups, and add the user and policy to that group
-:::
```

Comment:
Should we remove the manual creation steps all together? Curious if that would be a blocker for any of our customers?

Reply:
We don't document manual steps in other guides. The reasons are:
- It's very hard to keep them up to date
- If you are familiar with your cloud provider, doing the same than the CLI in the console it's quite simple

Reply:
That makes sense. Yes, not worth the effort of keeping up with provider UI changes